builtin/credential/okta/path_users.go

```diff
@@ -0,0 +1,139 @@
+package okta
```

Comment:
My understanding with this file is that, like LDAP, you want to offer per-user group additions. In that case, per-user policy additions would be great too -- see https://github.com/hashicorp/vault/pull/2152